src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -115,8 +120,23 @@ function PointsHistoryBottomSheet({ forwardedRef }: Props) {
         />
       </View>
     ) : (
-      <View testID={'PointsHistoryBottomSheet/Empty'}></View>
-    ) // TODO: Render empty/no history state
+      <View testID={'PointsHistoryBottomSheet/Empty'} style={styles.emptyContainer}>
+        <View style={styles.messageContainer}>
+          <Text style={styles.messageTitle}>{t('points.history.empty.title')}</Text>
+          <Text style={styles.messageSubtitle}>{t('points.history.empty.subtitle')}</Text>
+        </View>
+        <Button
+          testID={'PointsHistoryBottomSheet/LearnMore'}
+          onPress={onPressLearnMore}
+          text={t('points.history.empty.learnMore')}
+          type={BtnTypes.GRAY_WITH_BORDER}
+          size={BtnSizes.FULL}
+          style={{ width: '100%' }}
+        />
+      </View>
+    )
+
+  const renderEmpty = pointsHistoryStatus !== 'loading' && !pointsHistory.length
```

Comment:
nit: an attempt to make it slightly more self-explanatory
```suggestion
  const isEmpty = pointsHistoryStatus !== 'loading' && !pointsHistory.length
```